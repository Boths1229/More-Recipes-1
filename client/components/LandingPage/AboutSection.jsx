import React from 'react';
import img from '../../public/images/recipe-10.jpg';
import photo from '../../public/images/recipe-11.jpg';

const about = ' provides a platform for users to share the awesome and exciting recipe ideas they have invented or learnt. Suppose a user comes up with a food recipe, he/she can post it on More-Recipes and get feedback in form of reviews and votes from other users who explore that recipe. Users can also keep a list of their favorite recipes on the application.';

const AboutSection = () => (
  <div className="container">
    <section id="About" className="section-about">
      <div>
        <p>
          <span className="about">More-Recipes</span>
          { about }
        </p>
      </div>
      <div className="row">
        <div className="col-md-6 col-sm-3">
          <img className="img-fluid" src={img} alt="recipe" />
        </div>
        <div className="col-md-6 col-sm-3">
          <img className="img-fluid" src={photo} alt="recipe" />
        </div>
      </div>
    </section>
  </div>
);

export default AboutSection;
