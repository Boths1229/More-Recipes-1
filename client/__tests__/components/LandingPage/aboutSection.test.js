import React from 'react';
import { shallow } from 'enzyme';
import AboutSection from '../../../components/LandingPage/aboutSection';


describe('AboutSection Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<AboutSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
