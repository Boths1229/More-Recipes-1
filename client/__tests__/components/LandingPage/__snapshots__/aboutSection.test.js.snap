// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutSection Component should render correctly 1`] = `
<div
  className="container"
>
  <section
    className="section-about"
    id="About"
  >
    <div>
      <p>
        <span
          className="about"
        >
          More-Recipes
        </span>
         provides a platform for users to share the awesome and exciting recipe ideas they have invented or learnt. Suppose a user comes up with a food recipe, he/she can post it on More-Recipes and get feedback in form of reviews and votes from other users who explore that recipe. Users can also keep a list of their favorite recipes on the application.
      </p>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-6 col-sm-3"
      >
        <img
          alt="recipe"
          className="img-fluid"
          src="test"
        />
      </div>
      <div
        className="col-md-6 col-sm-3"
      >
        <img
          alt="recipe"
          className="img-fluid"
          src="test"
        />
      </div>
    </div>
  </section>
</div>
`;
